cmake/modules/StandaloneOverlay.cmake

```diff
@@ -58,9 +58,44 @@ set(SWIFT_NATIVE_CLANG_TOOLS_PATH "${TOOLCHAIN_DIR}/usr/bin" CACHE STRING
 set(SWIFT_NATIVE_SWIFT_TOOLS_PATH "${TOOLCHAIN_DIR}/usr/bin" CACHE STRING
   "Path to Swift tools that are executable on the build machine.")
 
+# HACK: This initialization code is duplicated in stdlib/CMakeLists.txt
+
+if("${SWIFT_HOST_VARIANT_SDK}" MATCHES "(OSX|IOS*|TVOS*|WATCHOS*)")
+  # All Darwin platforms have ABI stability.
+  set(SWIFT_STDLIB_STABLE_ABI_default TRUE)
+elseif("${SWIFT_HOST_VARIANT_SDK}" STREQUAL "LINUX")
```

Comment:
I'm pretty sure that this file is only used to build the overlays for official Apple platform releases, so we won't need this platform switch here -- this only ever applies to Darwin.

Reply:
Removed.